docs/python/python.rst

```diff
@@ -0,0 +1,170 @@
+=============
+Qt for Python
+=============
+
+.. sectionauthor:: `e8johan <https://github.com/e8johan>`_
+
+.. github:: ch106
+
+.. note::
+
+    Last Build: |today|
+
+    The source code for this chapter can be found in the `assets folder <../../assets>`_.
+
+
+This chapter ... TBD    
+
+Introduction
+============
+
+The Qt for Python project provides the tooling to bind C++ and Qt to Python, and a complete Python API to Qt. This means that everything that you can do with Qt and C++, you can also do with Qt and Python. This ranges from headless services to widget based user interfaces. In this chapter, we will focus on how to integrate QML and Python.
+
+Currently, Qt for Python is available for all desktop platforms, but not for mobile. Depending on which platform you use, the setup of Python is slightly different, but as soon as you have a `Python <https://www.python.org/>`_ and `PyPA <https://www.pypa.io/en/latest/>`_ environment setup, you can install Qt for Python using ``pip``. This is discussed in more detail further down.
+
+As the Qt for Python project provides an entirely new language binding for Qt, it also comes with a new set of documentation. The following resources are good to know about when exploring this module.
+
+- Reference documentation: `<https://doc.qt.io/qtforpython/>`_
+- Qt for Python wiki: `<https://wiki.qt.io/Qt_for_Python>`_
+- Caveats: `<https://wiki.qt.io/Qt_for_Python/Considerations>`_
+
+.. note::
+
+    Through-out this chapter we will use Python 3.6.
+
+Installing
+==========
+
+Qt for Python is available through PyPA using ``pip`` under the name ``pyside2``. In the example below we setup a ``venv`` environment in which we will install the latest version of Qt for Python:
+
+.. code-block:: bash
+
+    $ mkdir python-for-qt
```

Comment:
the other way around: `qt-for-python`, if not will cause more confusion.

Reply:
Good point :-)